addons/l10n_be/data/template/account.account-be.csv

```diff
@@ -210,31 +210,31 @@
 "a6095","Decrease (Increase) in Respect of Immovable Property for Resale","6095","expense_other","","False","Variations des stocks d'immeubles achetés destinés à la vente","Voorraadwijzigingen van gekochte onroerende goederen bestemd voor verkoop","Bestandsveränderungen der zum Verkauf bestimmten gekauften unbeweglichen Güter","",""
 "a61","Services and Other Goods","61","expense","account.account_tag_operating","False","Services et biens divers","Diensten en diverse goederen","Übrige Lieferungen und Leistungen","",""
 "a6101","Rent","6101","expense","","False","Louer","Huur","Mieten","Rental fees related to your workplace","attn_VAT-IN-V82-21-S"
-"a6102","Rental (cars)","6102","expense","","False","Location (voitures)","Verhuur (auto's)","Vermietung (Autos)","Rental of cars, including shared services, but excluding leasing",""
+"a6102","Rental (cars)","6102","expense","","False","Location (voitures)","Verhuur (auto's)","Vermietung (Autos)","Rental of cars, including shared services, but excluding leasing","attn_VAT-IN-V82-D35"
```

Comment:
Since we use a VAT deductibility of 35% here, might as well use the same on all other car costs and investments? @chklop